src/tests/simulation/src/cluster.rs

```diff
@@ -109,8 +128,9 @@ impl Configuration {
             meta_nodes: 3,
             compactor_nodes: 2,
             compute_node_cores: 2,
-            etcd_timeout_rate: 0.0,
-            etcd_data_path: None,
+            per_session_queries: vec!["SET STREAMING_ENABLE_ARRANGEMENT_BACKFILL=false".into()]
```

Comment:
Should this be unnecessary, I suppose? It's `false` by default.

Reply:
Necessary for testing purpose, it is set to `true` by default, each time we want to test arrangement backfill in the entire test suite.

It is a workaround for the `scaling` cases, which I plan to tackle in future PRs.